hologram_dashboard/fetcher/hologram_data_source_availability_fetcher_test.cc

```diff
@@ -64,6 +64,136 @@ TEST(FetcherTest, ValidAcquireConfig) {
         (expected_hologram_config, hologram_fetcher.hologram_configs_));
 }
 
+TEST(FetcherTest, UpdateHistoryNewProto) {
+    HologramDataSourceAvailabilityFetcher hologram_fetcher;
+    HologramDataAvailability hda;
+    absl::Time time = absl::Now();
+    StatusType status = StatusType::SUCCESS;
+    DataSourceDetail detail;
+    detail.set_date(absl::ToUnixSeconds(time));
+    detail.set_status(status);
+    hologram_fetcher.UpdateHistory(&hda, time, status);
+    EXPECT_EQ(hda.history_size(), 1);
+    EXPECT_TRUE(google::protobuf::util::MessageDifferencer::Equals(
+        hda.history(0), detail));
+}
+
+TEST(FetcherTest, UpdateHistorySameDay) {
+    HologramDataSourceAvailabilityFetcher hologram_fetcher;
+    HologramDataAvailability hda;
+    absl::Time time = absl::Now();
+    StatusType status = StatusType::SUCCESS;
+    DataSourceDetail detail;
+    detail.set_date(absl::ToUnixSeconds(time));
+    detail.set_status(status);
+    DataSourceDetail* history = hda.add_history();
+    history->set_date(absl::ToUnixSeconds(time));
+    history->set_status(status);
+    hologram_fetcher.UpdateHistory(&hda, time, status);
+    ASSERT_EQ(hda.history_size(), 1);
+    EXPECT_TRUE(google::protobuf::util::MessageDifferencer::Equals(
+        hda.history(0), detail));
+}
+
+TEST(FetcherTest, UpdateHistoryIncrementHistory) {
+    HologramDataSourceAvailabilityFetcher hologram_fetcher;
+    HologramDataAvailability hda;
+    StatusType status = StatusType::SUCCESS;
+    DataSourceDetail detail;
+    absl::CivilDay civil_time(2020, 5, 4);
+    absl::TimeZone google_time;
+    ASSERT_TRUE(absl::LoadTimeZone("America/Los_Angeles", &google_time));
```

Comment:
no need to assert check google time zone?

Reply:
I need it since according to the absl doc it sometimes may fail.